src/librustc/hir/mod.rs

```diff
@@ -355,30 +359,48 @@ pub struct PathParameters {
     pub parenthesized: bool,
```

Comment:
Does this field make sense without this struct being called `PathArgs` or something?

Reply:
Yes, the "parenthesized" property still applies to a single segment, not the whole path, e.g. `Fn(Args1)::Output<Args2>`.